### PR TITLE
add .gitattributes

### PR DESCRIPTION
**Pull request by jf17**
is not HTML library .

### CODE REVIEW

**Comment by jf17** [comment]
@googlebot I signed it!

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1 @@
+*.html linguist-language = Java
```
Comment (lyubomyr-shaydariv) [inline .gitattributes]: I think that mimicking all HTML files as Java source code is not a very good idea. According to the Linguist [documentation](https://github.com/github/linguist#documentation), the following line looks like a more correct way:

```
gson/docs/javadocs/* linguist-documentation
```

**Comment by jf17** [comment]
@lyubomyr-shaydariv look good, thanks!

**Comment by lyubomyr-shaydariv** [comment]
By the way,

* You don't need to have both 4790307eef5be79eff0bc7517754a4b4916134b2 and a6e6c61de10f15ed2272c2bfdcec7a38ddc74f79 commits. Could you please simply combine both commits into a single one (by amending the initial commit or rebasing the two using fixup/squash)?
* I would preserve the commit message from the second commit. I think that its ```Use the linguist-documentation attribute to mark paths as documentation.``` is more accurate.

**Pull request closed without merging**